tests/jvm/src/test/resources/metacp.expect

```diff
@@ -36,6 +36,422 @@ _empty_.B#`<init>`(). => primaryctor <init>: (): B
+  Unit => _root_.scala.Unit#
+_root_.com.javacp.Javacp#publicField. => var publicField: Int
+  Int => _root_.scala.Int#
+_root_.com.javacp.Javacp#publicMethod(). => def publicMethod: (): Unit
+  Unit => _root_.scala.Unit#
+_root_.com.javacp.Javacp#typeParams(ArrayList,Hashtable). => private[javacp] def typeParams: (arg0: ArrayList[HashMap[A, Array[String]]], arg1: Hashtable[String, B]): Unit
+  arg0 => _root_.com.javacp.Javacp#typeParams(ArrayList,Hashtable).(arg0)
+  arg1 => _root_.com.javacp.Javacp#typeParams(ArrayList,Hashtable).(arg1)
+  Unit => _root_.scala.Unit#
+_root_.com.javacp.Javacp#typeParams(ArrayList,Hashtable).(arg0) => param arg0: ArrayList[HashMap[A, Array[String]]]
+  ArrayList => _root_.java.util.ArrayList#
+  HashMap => _root_.java.util.HashMap#
+  A => _root_.com.javacp.Javacp#[A]
+  Array => _root_.scala.Array#
+  String => _root_.java.lang.String#
+_root_.com.javacp.Javacp#typeParams(ArrayList,Hashtable).(arg1) => param arg1: Hashtable[String, B]
+  Hashtable => _root_.java.util.Hashtable#
+  String => _root_.java.lang.String#
+  B => _root_.com.javacp.Javacp#[B]
+_root_.com.javacp.Javacp#wildcard(ArrayList). => def wildcard: (arg0: ArrayList[local_wildcard]): Unit
+  arg0 => _root_.com.javacp.Javacp#wildcard(ArrayList).(arg0)
+  Unit => _root_.scala.Unit#
+_root_.com.javacp.Javacp#wildcard(ArrayList).(arg0) => param arg0: ArrayList[local_wildcard]
+  ArrayList => _root_.java.util.ArrayList#
+  local_wildcard => local_wildcard
+_root_.com.javacp.Javacp#wildcard(Collection,Collection). => def wildcard: (arg0: Collection[Number], arg1: Collection[Number]): Unit
+  arg0 => _root_.com.javacp.Javacp#wildcard(Collection,Collection).(arg0)
+  arg1 => _root_.com.javacp.Javacp#wildcard(Collection,Collection).(arg1)
+  Unit => _root_.scala.Unit#
+_root_.com.javacp.Javacp#wildcard(Collection,Collection).(arg0) => param arg0: Collection[Number]
+  Collection => _root_.java.util.Collection#
+  Number => _root_.java.lang.Number#
+_root_.com.javacp.Javacp#wildcard(Collection,Collection).(arg1) => param arg1: Collection[Number]
+  Collection => _root_.java.util.Collection#
+  Number => _root_.java.lang.Number#
+
+com/javacp/NonGeneric.class
+---------------------------
+
+Summary:
+Schema => SemanticDB v3
+Uri => com/javacp/NonGeneric.class
+Text => empty
+Language => Java
+Symbols => 6 entries
+
+Symbols:
+_root_. => package _root_
+_root_.com. => package com
+_root_.com.javacp. => package javacp
+_root_.com.javacp.NonGeneric# => class NonGeneric.{+2 decls}
+  extends Exception
+  extends Serializable
+  extends Flushable
+_root_.com.javacp.NonGeneric#<init>(). => def <init>: (): Unit
+  Unit => _root_.scala.Unit#
+_root_.com.javacp.NonGeneric#flush(). => def flush: (): Unit
+  Unit => _root_.scala.Unit#
+
+com/javacp/Recursive$1.class
+----------------------------
+
+Summary:
+Schema => SemanticDB v3
+Uri => com/javacp/Recursive$1.class
+Text => empty
+Language => Java
```

Comment:
Do you think we should have SemanticDB entries for anonymous classes? We don't in Scala.

Reply:
Done.